journals/apps/api/v1/content/views.py

```diff
@@ -0,0 +1,22 @@
+"""
+Overridden Wagtail API endpoints
+"""
+from rest_framework.permissions import AllowAny
+from rest_framework.exceptions import PermissionDenied
+from wagtail.api.v2.endpoints import PagesAPIEndpoint
+from wagtail.wagtailcore.models import Page
+# from wagtail.wagtailcore.utils import resolve_model_string
+
+from journals.apps.journals.models import JournalAccess, JournalPage
+from journals.apps.api.filters import PageAuthorizationFilter
+
+class FilteredPagesAPIEndpoint(PagesAPIEndpoint):
```

Comment:
Would rather name this something like `JournalPagesAPIEndpoint` as we're likely to extend it to do things other than filtering and would be consistent with how we override other wagtail classes.

Reply:
Fixed